### PR TITLE
misc Orchestration cleanup

### PR DESCRIPTION
**Pull request by turadg**
_incidental_

## Description
Miscellaneous cleanups while starting https://github.com/Agoric/agoric-sdk/pull/9719 for #9303.

See each commit title for changes

### Security Considerations
none

### Scaling Considerations
none

### Documentation Considerations
none

### Testing Considerations
CI

### Upgrade Considerations
not yet deployed

### CODE REVIEW

File: packages/orchestration/src/typeGuards.js [hunk 1]
```diff
@@ -35,9 +35,6 @@ export const Proto3Shape = {
   value: M.string(),
 };
 
-// XXX same as ChainAmountShape and DenomAmount type
-export const CoinShape = { value: M.bigint(), denom: M.string() };
```
Comment (dckc) [inline packages/orchestration/src/typeGuards.js:39]: :+1:

File: packages/orchestration/src/facade.js [hunk 2]
```diff
@@ -14,6 +14,19 @@ import { assertAllDefined } from '@agoric/internal';
...
+ * For a given guest passed to orchestrate(), return the host-side form.
+ *
+ * @template {(orc: Orchestrator, ctx: any, ...args: any[]) => Promise<any>} GF
+ * @typedef {GF extends (
+ *   orc: Orchestrator,
+ *   ctx: any,
+ *   ...args: infer GA
+ * ) => Promise<infer GR>
+ *   ? (...args: HostArgs<GA>) => Vow<GR>
+ *   : never} HostForGuest
```
Comment (dckc) [inline packages/orchestration/src/facade.js:27]: this is at the edge of my tsc-fu; an example in the form of a type test would help me.

**Review by dckc — APPROVED** [review]
good stuff.

one suggestion

File: packages/orchestration/src/facade.js (hunk 2 above)
Comment (turadg) [reply packages/orchestration/src/facade.js:27]: worthwhile. probably another PR.

**Pull request merged**